src/authenticationMiddleware.test.ts

```diff
@@ -11,80 +16,81 @@ test("required returns correct values", (t) => {
     [false, "boolean"],
     [true, "boolean"],
   ];
-  tests.forEach(([expected, name]) => {
+  for (const [expected, name] of cases) {
     const actual = required(expected, name);
     t.deepEqual(actual, expected);
-  });
+  }
 });
 
 test("validateAudience passes on valid values", (t) => {
   const tests: string[] = [
     "/projects/000000000000/global/backendServices/0000000000000000000",
     "/projects/000000000000/apps/my-sample-project-191923",
   ];
-  tests.forEach(audience => {
```

Comment:
why not `.forEach`?

Reply:
I think I had to do this when I tried to update `ava`, but it's no longer part of this PR, so I reverted the change. Thank you.